cmd/loop/loopin.go

```diff
@@ -157,6 +157,7 @@ func loopIn(ctx *cli.Context) error {
 		fmt.Printf("HTLC address (NP2WSH): %v\n", resp.HtlcAddressNp2Wsh)
 	}
 	fmt.Printf("HTLC address (P2WSH): %v\n", resp.HtlcAddressP2Wsh)
+	fmt.Printf("Server message: %v\n", resp.ServerMessage)
```

Comment:
This will always print. How about printing only if `strings.Trim(resp.ServerMessage, " ") != ""`.

Reply:
Added

Reply:
Removed trim part again. It is a bit ugly. I think I will do a follow up pr on the server to clear after all, just in case we want to get rid of it quickly without any trace.